extensions/interactions/MathEquationInput/directives/math-equation-input-rules.service.ts

```diff
@@ -36,7 +37,7 @@ import {
 export class MathEquationInputRulesService {
   MatchesExactlyWith(
       answer: MathEquationAnswer,
-      inputs: MathEquationMatchesExactlyWithRuleInputs): boolean {
+      inputs: MathEquationRuleInputsWithSide): boolean {
```

Comment:
What do you mean by 'withSide'?

Reply:
The `IsEquivalentTo` rule input doesn't require the `y` parameter which denotes the side of the equation which is relevant for the rule check. This is why we need different typedefs for this interaction, ie, one with side and the other without. One with placeholder is also added in this PR for the general match rule.

Reply:
Ah ok, maybe add a comment where it is defined then (if that isn't there)?

Reply:
Done.